src/index.html

```diff
@@ -21,7 +21,368 @@
+        </div>
+
+        <a class="card__button">Buy</a>
+      </div>
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="image iMac"
+          class="card__img"
+        />
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__reviews">
+          <div class="card__reviews_stars">
+            <div class="stars stars--1"></div>
+            <div class="stars stars--2"></div>
+            <div class="stars stars--3"></div>
+            <div class="stars stars--4"></div>
+            <div class="stars stars--5"></div>
+          </div>
+          <div class="card__reviews_count">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price_title">Price:</div>
+          <div class="card__price_sum"></div>
+        </div>
+
+        <a class="card__button">Buy</a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="image iMac"
+          class="card__img"
+        />
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__reviews">
+          <div class="card__reviews_stars">
+            <div class="stars stars--1"></div>
+            <div class="stars stars--2"></div>
+            <div class="stars stars--3"></div>
+            <div class="stars stars--4"></div>
+            <div class="stars stars--5"></div>
+          </div>
+          <div class="card__reviews_count">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price_title">Price:</div>
+          <div class="card__price_sum"></div>
+        </div>
+
+        <a class="card__button">Buy</a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="image iMac"
+          class="card__img"
+        />
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__reviews">
+          <div class="card__reviews_stars">
+            <div class="stars stars--1"></div>
+            <div class="stars stars--2"></div>
+            <div class="stars stars--3"></div>
+            <div class="stars stars--4"></div>
+            <div class="stars stars--5"></div>
+          </div>
+          <div class="card__reviews_count">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price_title">Price:</div>
+          <div class="card__price_sum"></div>
```

Comment:
The 'card__price_sum' div is empty. Ensure that this is intentional or populate it with the appropriate price information.